BookReaderDemo/demo-internetarchive.html

```diff
@@ -2,70 +2,74 @@
-    <script src="../BookReader/mmenu/dist/js/jquery.mmenu.min.js"></script>
-    <script src="../BookReader/mmenu/dist/addons/navbars/jquery.mmenu.navbars.min.js" ></script>
-    <link rel="stylesheet" href="../BookReader/mmenu/dist/css/jquery.mmenu.css" />
-    <link rel="stylesheet" href="../BookReader/mmenu/dist/addons/navbars/jquery.mmenu.navbars.css" />
-
-    <!-- Bookreader -->
-    <script src="../BookReader/BookReader.js"></script>
-    <link rel="stylesheet" href="../BookReader/BookReader.css"/>
-
-    <!-- plugins needed for archive.org, in same order as archive.org -->
-    <script src="../BookReader/plugins/plugin.mobile_nav.js"></script>
-    <script src="../BookReader/plugins/plugin.search.js"></script>
-    <script src="../BookReader/plugins/plugin.chapters.js"></script>
-    <script src="../BookReader/plugins/plugin.tts.js"></script>
-    <script src="../BookReader/plugins/plugin.url.js"></script>
-    <script src="../BookReader/plugins/plugin.autoplay.js"></script>
-    <script src="../BookReader/plugins/plugin.resume.js"></script>
-    <script src="../BookReader/plugins/plugin.archive_analytics.js"></script>
-    <script src="../BookReader/plugins/plugin.text_selection.js"></script>
-
-    
-    <link rel="stylesheet" href="BookReaderDemo.css"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+  <meta name="apple-mobile-web-app-capable" content="yes">
+
+  <!-- JS dependencies -->
+  <script src="../BookReader/jquery-1.10.1.js"></script>
+
+
+  <!-- Bookreader -->
+  <script src="../BookReader/BookReader.js"></script>
+
+  <link rel="stylesheet" href="../BookReader/BookReader.css" />
+
+  <!-- plugins needed for archive.org, in same order as archive.org -->
+  <script src="../BookReader/plugins/plugin.mobile_nav.js"></script>
+  <script src="../BookReader/plugins/plugin.search.js"></script>
+  <script src="../BookReader/plugins/plugin.chapters.js"></script>
+  <script src="../BookReader/plugins/plugin.tts.js"></script>
+  <script src="../BookReader/plugins/plugin.url.js"></script>
+  <script src="../BookReader/plugins/plugin.autoplay.js"></script>
+  <script src="../BookReader/plugins/plugin.resume.js"></script>
+  <script src="../BookReader/plugins/plugin.archive_analytics.js"></script>
+  <script src="../BookReader/plugins/plugin.text_selection.js"></script>
+
+  <!-- CSS only -->
+  <link rel="stylesheet" href="BookReaderDemo.css" />
+
+  <link rel="stylesheet" href="../BookReaderTemplate/styles/BookReader-ia.css" />
+
 </head>
+
 <body>
+  <div id="theatre-ia">
+    <div class="row" style="background-color: black; ">
 
-<div id="BookReader">
-    Internet Archive BookReader Demo<br/>
-    <noscript>
-    <p>
-        The BookReader requires JavaScript to be enabled. Please check that your browser supports JavaScript and that it is enabled in the browser settings.
-    </p>
-    </noscript>
-</div>
+      <div id="bookreader-container" class="xs-col-12">
```

Comment:
`xs-col-12` looks like boostrap; I don't think this will work in BR? We might be able to remove it.

Reply:
removed this one